mock/py/mockbuild/config.py

```diff
@@ -83,6 +83,7 @@ def setup_default_config_opts():
     config_opts['use_bootstrap'] = True
     config_opts['use_bootstrap_image'] = False
     config_opts['bootstrap_image'] = 'fedora:latest'
+    config_opts['bootstrap_image_ready'] = False
```

Comment:
Can this be documented in site-defaults.cfg too?

Reply:
Truth, thank you for pointing that out!